src/client/sandbox/event/message.js

```diff
@@ -207,8 +207,7 @@ export default class MessageSandbox extends SandboxBase {
                     // trying to call the 'isCrossDomainWindows' function, so we have to wrap it in 'try/catch'.
                     try {
                         targetWindow[this.RECEIVE_MSG_FN]({
-                            // NOTE: Cloning a message to prevent this modification.
```

Comment:
Are all tests are ok with this? I suppose it should be something sensitive since it was especially commented